rmm/src/rmi/realm/params.rs

```diff
@@ -67,17 +44,17 @@ impl Hashable for Params {
     ) -> Result<(), crate::measurement::MeasurementError> {
         hasher.hash_fields_into(out, |alg| {
             alg.hash_u64(0); // features aren't used
-            alg.hash(self.padding0);
+            alg.hash(self._padfeatures_0);
```

Comment:
Could we use the origin name of the field? I think we should keep this for intuitive code.

```suggestion
            alg.hash(self.features_0);
```

Reply:
I guess it's meaningless to hash padded fields here. Removed all padded fields here with new commit :)